install/kubernetes/helm/istio/charts/prometheus/templates/gateway.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.ingressgateway.enabled -}}
+{{- $serviceName := include "prometheus.fullname" . -}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: prometheus-gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
```

Comment:
Not on port 80 please.

Reply:
Use 150xx range if possible, so it's easy to set firewall rules

Reply:
+1. No port 80. I don't want to debug user issues with their stuff on port 80 and ours as well on port 80. Not to mention, this gets exposed to the web

Reply:
Well - that's the intent of this PR ( to fix the previous "Ingress" which is no longer enabled by default, but it was exposing on the web ). Not on 80, but still public.